test/executor.test.js

```diff
@@ -468,9 +468,25 @@ describe('executor', function() {
 
     boot(app, appdir.PATH);
 
+    expect(containsSubstring(Object.keys(require.cache),
+      'components\\test-component\\index.js')).to.eql(true);
```

Comment:
1) This will not work on unix, where the path separator is `\`.
2) It's possible that multiple tests are using the same relative path, in which case this check will incorrectly pass. A better solution is to use build a full path using `sandbox.resolve('components', 'test-componen', 'index.js')` and the assert that the keys contain this entry (there is a chai check for that).